Simplenote/Supporting Files/Simplenote-Info.plist

```diff
@@ -75,6 +75,7 @@
 	<key>NSUserActivityTypes</key>
 	<array>
 		<string>ListWidgetIntent</string>
+		<string>NoteWidgetIntent</string>
 		<string>SPNoteWidgetIntent</string>
```

Comment:
```suggestion

```

I'm pretty sure that SPNoteWidgetIntent is an earlier version of NoteWidgetIntent.  With the code generated intents files, by default the system gives the class a name and adds the app prefix.  `NoteWidgetIntent` should be the custom class that replaces `SPNoteWidgetIntent` so there is no prefix.  

We should be able to drop this `SPNoteWidgetIntent` here